torch2trt/torch2trt.py

```
@@ -735,8 +748,10 @@ def tensorrt_converter(method, is_real=True, enabled=True, imports=[]):
 
     try:
         method_impl = eval('copy.deepcopy(module.%s)' % qual_name)
-    except:
+    except (AttributeError, NameError):
         enabled = False
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)